src/OmniSharp.Host/Services/OmniSharpEnvironment.cs

```diff
@@ -38,6 +40,20 @@ public class OmniSharpEnvironment : IOmniSharpEnvironment
             TraceType = traceType;
             TransportType = transportType;
             OtherArgs = otherArgs;
+
+            // On Windows: %APPDATA%\OmniSharp\omnisharp.json
+            // On Mac/Linux: ~/.omnisharp/omnisharp.json
+            var root = Environment.GetEnvironmentVariable("APPDATA") ??
+                Environment.GetEnvironmentVariable("HOME");
+
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("APPDATA")))
```

Comment:
Nit: It seems a little funny that we get the environment variable for APPDATA and then get it again immediately.